window/src/os/windows/window.rs

```diff
@@ -716,72 +735,129 @@ impl WindowOps for Window {
         clipboard_win::set_clipboard_string(&text).ok();
     }
 
-    fn get_title_font_and_point_size(&self) -> Option<(wezterm_font::parser::ParsedFont, f64)> {
-        const TMT_CAPTIONFONT: i32 = 801;
-        const HP_HEADERITEM: i32 = 1;
-        const HIS_NORMAL: i32 = 1;
-
-        unsafe fn populate_log_font(hwnd: HWND, hdc: HDC) -> Option<LOGFONTW> {
-            let mut log_font = LOGFONTW {
-                lfHeight: 0,
-                lfWidth: 0,
-                lfEscapement: 0,
-                lfOrientation: 0,
-                lfWeight: 0,
-                lfItalic: 0,
-                lfUnderline: 0,
-                lfStrikeOut: 0,
-                lfCharSet: 0,
-                lfOutPrecision: 0,
-                lfClipPrecision: 0,
-                lfQuality: 0,
-                lfPitchAndFamily: 0,
-                lfFaceName: [0u16; 32],
-            };
-            let theme = OpenThemeData(
-                hwnd,
-                [
-                    'H' as u16, 'E' as u16, 'A' as u16, 'D' as u16, 'E' as u16, 'R' as u16, 0u16,
-                ]
-                .as_ptr(),
-            );
-            if !theme.is_null() {
-                let res = GetThemeFont(
-                    theme,
-                    hdc,
-                    HP_HEADERITEM,
-                    HIS_NORMAL,
-                    TMT_CAPTIONFONT,
-                    &mut log_font,
-                );
-                if res == S_OK {
-                    CloseThemeData(theme);
-                    return Some(log_font);
-                }
-            }
+    fn get_os_parameters(&self, config: &ConfigHandle) -> anyhow::Result<Option<Parameters>> {
+        let hwnd = self.0 .0;
+        if hwnd.is_null() {
+            return Err(anyhow::anyhow!("HWND is null"));
+        }
 
-            let res = GetThemeSysFont(theme, TMT_CAPTIONFONT, &mut log_font);
-            if !theme.is_null() {
-                CloseThemeData(theme);
-            }
+        let has_focus = !unsafe { GetFocus() }.is_null();
```

Comment:
I think this may produce incorrect results when there are multiple windows.  How about this instead?

```suggestion
        let has_focus = unsafe { GetFocus() } == hwnd;
```